src/storage/v2/inmemory/storage.cpp

```diff
@@ -1958,7 +1969,14 @@ void InMemoryStorage::CollectGarbage(std::unique_lock<utils::ResourceLock> main_
       guard.unlock();
       // if lucky, there are no active transactions, hence nothing looking at the deltas
       // remove them now
+
+      size_t total_delta_size = 0;
+      for (const auto &gc_deltas : unlinked_undo_buffers) {
+        total_delta_size += gc_deltas.deltas_.size();
+      }
```

Comment:
```suggestion
     auto const sum_func = [](uint32_t curr, auto const &gc_deltas) {
       return curr + gc_deltas.deltas_.size();
      }

      uint32_t total_delta_size = std::accumulate(unlinked_undo_buffers.begin(), unlinked_undo_buffers.end(), 0U, sum_func);
    
```
This should be correct, maybe I have some typo